pyproject.toml

```diff
@@ -38,6 +38,8 @@ dynamic = ["version"]
 
 [project.optional-dependencies]
 dev = [
+    "black",
+    "clang-format==15.0.7",
```

Comment:
I would not put here. Let's install clang-format from `apt` or Visual Studio. Not via pip.

Reply:
ok, will remove again